tests/expected/raw_slice/slice.rs

```diff
@@ -0,0 +1,102 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// kani-flags: --unwind 3
+
+//! This test case has a bunch of checks related to structures using raw slices ([T]).
+use std::mem;
+
+/// Non-sized structure with a sized element and an unsized element.
+struct NonEmptySlice {
```

Comment:
I don't think we need it, but should we see what happens with different repr here?

Reply:
Do you mean add a harness that checks the representation?